ui/app/components/clients/page/counts.ts

```diff
@@ -82,6 +82,12 @@ export default class ClientsCountsPageComponent extends Component<Args> {
     return null;
   }
 
+  // passed into page-header for the export modal alert
+  get upgradesDuringActivity() {
```

Comment:
Now that this getter lives here, should we replace line 58 with it in `upgradeExplanations` above?

https://github.com/hashicorp/vault/blob/266ea693ccdec27718c17b7ee354cc6e19d9f305/ui/app/components/clients/page/counts.ts#L56-L59